public/app/plugins/datasource/elasticsearch/specs/elastic_response.test.ts

```diff
@@ -210,22 +230,52 @@ describe('ElasticResponse', () => {
       };
 
       result = new ElasticResponse(targets, response).getTimeSeries();
-    });
-
-    it('should return 2 series', () => {
       expect(result.data.length).toBe(4);
-      expect(result.data[0].datapoints.length).toBe(2);
-      expect(result.data[0].target).toBe('server1 Count');
-      expect(result.data[1].target).toBe('server1 Average @value');
-      expect(result.data[2].target).toBe('server2 Count');
-      expect(result.data[3].target).toBe('server2 Average @value');
+
+      expect(result.data[0].length).toBe(2);
```

Comment:
It's probably better to use `expect(result.data[0]).toEqual({name: 'server1 Count', field: [...]...})` and check it with one expect call.